proto/sentry_protos/snuba/v1/endpoint_trace_item_columns.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TraceItemColumnsRequest {
```

Comment:
What is this endpoint for?

Reply:
This endpoint is to serve table views

Reply:
Would naming it `TraceItemTableRequest` perhaps be better? Might not be as abstract but communicates the intent better

Reply:
Sounds good